tools/remote_build/windows.bazelrc

```diff
@@ -47,7 +47,7 @@ build --bes_timeout=60s
 build --bes_results_url="https://source.cloud.google.com/results/invocations/"
 build --project_id=grpc-testing
 
-build --jobs=30
+build --jobs=100
```

Comment:
I increased the number of RBE windows workers in our pool (50->200), so increasing the number of parallel jobs here as well.